File: edu/HRO/CMITDW Taal der Wiskunde/Lua/o21.lua
-- opdracht 2.1; Factorial
local function factorial(n)
	local sum = 1
	for i = 2, n do
		sum = sum * i
	end
	return sum
end

for i = 1, 200 do -- overflow vanaf 171
	print(i, factorial(i))
end

File: HRO/CMITDW Taal der Wiskunde/o21.java
// opdracht 2.1; Factorial
public class o21
{
	public static void main(String[] args)
	{
		for (int i=0; i<=200; i++) // overflow after 171
		{
			System.out.println(i+"\t"+factorial(i));
		}
	}
	
	public static double factorial(int n)
	{
		double sum = 1;
		for (int i=2; i<=n; i++)
			sum *= i;
		return sum;
	}
}

